webroot/adm/client-side-keypairs.html

```diff
@@ -93,20 +93,35 @@ <h3>Output</h3>
             const name = $('#name').val();
             const disabled = ($('#disable').is(':checked'));
 
-            if(!subscriptionId) {
+            if (!subscriptionId) {
                 $('#errorOutput').text("required parameters: subscription_id")
                 return
             }
 
+            const confirmationMessage = getUpdateKeypairConfirmationMessage(disabled, subscriptionId);
+            if (!confirm(confirmationMessage)) return;
+
             const payload = {"subscription_id": subscriptionId, "disabled" : disabled}
-            if(name){
+            if (name) {
                 payload["name"] = name
             }
 
             doApiCall('POST', '/api/client_side_keypairs/update', '#standardOutput', '#errorOutput', JSON.stringify(payload));
         });
 
     });
+
+    function getUpdateKeypairConfirmationMessage(disabled, subscriptionId) {
+        let action = disabled ? 'disable' : 'enable';
+        let confirmationMessage = `You may ignore this warning if you are not changing the disabled status of this keypair.\n\nAre you sure you want to ${action} ${subscriptionId}?`;
```

Comment:
The pattern for client-side-keypairs is a little different, since it just has a generic update function which is given a disabled flag, rather than explicit `enable` and `disable` buttons. Ideally this would line would not be necessary, but I'm not sure how much effort we should put into making this better right now.